### PR TITLE
Updated cockpit guide to start cockpit

### PR DESCRIPTION
**Pull request by brgnepal**
Updated as a result of following PR change in Vagrant service manager plugin.
https://github.com/projectatomic/vagrant-service-manager/pull/219

### CODE REVIEW

File: docs/cockpit.rst [hunk 1]
```diff
@@ -23,19 +23,19 @@ the host you are running the ADB on.
 
 1. Start the ADB
 
-2. Start cockpit
+2. Start Cockpit
 
-   ``$ vagrant service-manager restart cockpit``
+   ``$ vagrant service-manager start cockpit``
```
Comment (hferentschik) [inline docs/cockpit.rst]: I am not opposed to a `service-manager start/stop`, but still don't like that we use it to control cockpit. For now, I think we would be better of to use `systemctl` for that.

**Comment by brgnepal** [comment]
@praveenkumar @bexelbie @LalatenduMohanty 

Since I have reverted all the changes related to `systemd` services in this PR https://github.com/projectatomic/vagrant-service-manager/pull/219. I would better suggest here to go with `systemctl` approach .

**Comment by praveenkumar** [comment]
@budhrg LGTM

**Comment by bexelbie** [comment]
LGTM

**Pull request merged**